docs/instructions.md

```diff
@@ -34,6 +34,65 @@ TRAIN_FUNCTION=<function to invoke to start model training>
 10. to run the predict container:
 > `docker run --rm  -v $(PROJECT_ROOT_DIR)/models:/action/models -e MODElNAME=${MODEL_NAME} -p 5111:5000 -it ${PREDICT_CONTAINER_NAME}`
 
+## Model Predict: Request/Response
+
+The predict endpoint takes an array of instances containing an array of feature values.
+```
+curl -X POST \
+ http://localhost:5111/<route> \
+ -H 'Content-Type: application/json' \
+ -d '{
+"payload": { "instances": [[7,107,74,0,0,29.6,0.254,31]] }
+}'
+```
+`route` is defined using .s2i/environment file
+
+It returns an array of predictions.
+```
+{
+   "payload": {
+     "predictions": [1]
+   }
+}
+```
+
+Sample example predict for the bank_marketing:
+
+`REQUEST`
+
+```
+curl -X POST \
+  http://localhost:5111/bank_marketing_logit/predict  \
+  -H 'Content-Type: application/json' \
+  -d '{
+      "payload": {
```

Comment:
should be an array of arrays to avoid confusion. OR we have to add text to explain that instances can also be a single instance.

Reply:
> Think this is already in master...?

yes, this is merged. Let me fix that